examples/examples/jsonrpsee_server_disconnect_conn.rs

```diff
@@ -0,0 +1,264 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
//cc @xlc 

This is the suggested API to disconnect peers etc, it's still just a PoC and I need to polish it.
Is it sufficient for proxy implementation?

This example just disconnect that don't comply with the "dummy rate limit middleware" and blacklists the IP addr.